### PR TITLE
HiDPI Fixes

### PR DESCRIPTION
**Pull request by q234rty**
Fix various downscaled icons by removing undeeded multiplications by devicePixelRatio in sourceSize.

Fix downscaled PL indicator in the timeline by using the actual preferred size as sourceSize.

Fix various blurry icons by specifying sourceSize.

Note that this PR does not change the actual design in any way, those will come as seperate PRs.

### CODE REVIEW

File: resources/qml/TimelineSectionHeader.qml [hunk 1]
```diff
@@ -92,8 +92,8 @@ Column {
                 height: fontMetrics.ascent
                 width: height
 
-                sourceSize.width: fontMetrics.lineSpacing
-                sourceSize.height: fontMetrics.lineSpacing
+                sourceSize.width: fontMetrics.ascent
+                sourceSize.height: fontMetrics.ascent
```
Comment (nico-famedly) [inline resources/qml/TimelineSectionHeader.qml:96]: You should probably just set this to height and width (also in the other places), but I guess this is fine for now.

File: resources/qml/dialogs/RoomMembers.qml [hunk 2]
```diff
@@ -162,6 +162,10 @@ ApplicationWindow {
                         }
 
                         PowerlevelIndicator {
+                            Layout.preferredWidth: 32
```
Comment (nico-famedly) [inline resources/qml/dialogs/RoomMembers.qml:165]: I know you didn't want to change the design, but this really shouldn't be 2x as big as the other one. If we are spelling out a size for this now, can we make both of them 2x the line height?

Comment (nico-famedly) [reply resources/qml/dialogs/RoomMembers.qml:165]: Although I guess that won't work for your own profile?

**Comment by deepbluev7** [comment]
@labhub rerun

**Comment by deepbluev7** [comment]
Sorry, but I don't know what that command means.

Thanks for asking 🥰

**Comment by deepbluev7** [comment]
@labhub retry

File: resources/qml/TimelineSectionHeader.qml (hunk 1 above)
Comment (q234rty) [reply resources/qml/TimelineSectionHeader.qml]: Set those to height/width in various places, haven't found a binding loop after some testing.

File: resources/qml/dialogs/RoomMembers.qml (hunk 2 above)
Comment (q234rty) [reply resources/qml/dialogs/RoomMembers.qml]: Made them both `fontMetrics.lineSpacing * 2`.

**Comment by deepbluev7** [comment]
Looks good to me, thanks!

**Pull request merged**